samples/01.getting-started/l.sharepoint-web-part/README.md

```diff
@@ -1,3 +1,5 @@
+> 💥 The sample has been archived and will no longer receive updates.
```

Comment:
Do we have a plan for removal/replacement? Something always using latest and we have a e2e test checking if we failed it?

Reply:
No plan for exact replacement especially for SharePoint. If we go Web Components, then it will be the true replacement. 🙂